### PR TITLE
Worker executor and some small data fixer related mappings.

### PR DESCRIPTION
**Pull request by i509VCB**

### CODE REVIEW

File: mappings/net/minecraft/util/Util.mapping [hunk 1]
```diff
@@ -3,11 +3,13 @@ CLASS net/minecraft/class_156 net/minecraft/util/Util
 	FIELD field_1129 LOGGER Lorg/apache/logging/log4j/Logger;
 	FIELD field_18034 NEXT_SERVER_WORKER_ID Ljava/util/concurrent/atomic/AtomicInteger;
 	FIELD field_18035 SERVER_WORKER_EXECUTOR Ljava/util/concurrent/ExecutorService;
```
Comment (liach) [inline mappings/net/minecraft/util/Util.mapping]: This and the next server worker id have to be fixed.

File: mappings/net/minecraft/util/Util.mapping [hunk 2]
```diff
@@ -9,7 +9,7 @@ CLASS net/minecraft/class_156 net/minecraft/util/Util
 	METHOD method_17815 stream (Ljava/util/Optional;)Ljava/util/stream/Stream;
 	METHOD method_17974 ifPresentOrElse (Ljava/util/Optional;Ljava/util/function/Consumer;Ljava/lang/Runnable;)Ljava/util/Optional;
 	METHOD method_18349 getServerWorkerExecutor ()Ljava/util/concurrent/Executor;
```
Comment (liach) [inline mappings/net/minecraft/util/Util.mapping]: I hope you can fix this one and the fields as well.

Comment (i509VCB) [reply mappings/net/minecraft/util/Util.mapping]: The name?

**Review by jaskarth — APPROVED** [review]

**Review by liach — CHANGES_REQUESTED** [review]
If you fix executor names, should fix all executor names at once than just fixing one method.

**Review by liach — APPROVED** [review]
Looks good

**Review by YanisBft — APPROVED** [review]

**Pull request merged**